pkg/blobstore/configuration/proto_blob_replicator_creator.go

```diff
@@ -14,3 +14,7 @@ type protoBlobReplicatorCreator struct{}
 func (brc protoBlobReplicatorCreator) NewCustomBlobReplicator(configuration *pb.BlobReplicatorConfiguration, source blobstore.BlobAccess, sink BlobAccessInfo) (replication.BlobReplicator, error) {
 	return nil, status.Error(codes.InvalidArgument, "Configuration did not contain a supported replicator")
 }
+
+func (brc protoBlobReplicatorCreator) GetStorageTypeName() string {
```

Comment:
Please remove this, as it is not valid to return an empty string.  Any type that is based on `protoBlobReplicatorCreator` should bring its own implementation.